miio/chuangmi_ir.py

```diff
@@ -130,10 +130,10 @@ def play(self, command: str):
 
 
 class ProntoPulseAdapter(Adapter):
-    def _decode(self, obj, context):
+    def _decode(self, obj, context, path):
```

Comment:
If the code doesn't use the new path arg, wouldn't it be better to add here `*args, **kwargs` to just ignore any extra arguments? It would make it backward compatible with earlier Construct versions.

Reply:
Nice approach! I updated the PR.